src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -54,6 +55,7 @@ limitations under the License.
     <define>OPERATIONAL_CREDENTIALS_CLUSTER</define>
     <description>This cluster is used to add or remove Operational Credentials on a Commissionee or Node, as well as manage the associated Fabrics.</description>
 
+    <attribute side="server" code="0x0000" define="NOCS" type="ARRAY" entryType="NOCStruct" length="254" writable="false" optional="false">NOCs</attribute>
```

Comment:
Please leave the `length` attribute out.  It's meaningless, and 254 bytes would not be useful anyway.

Reply:
If length="254" does not represent the maximum bytes allows in the ARRAY element, what is the real limitation?

Reply:
On the implementation side?  For writable attributes implemented via AttributeAccessInterface, whatever checks it does on write.  For reads, whatever data it produces.

On the spec side, the limit is "max SupportedFabrics" for the list _length_.  So the amount of data is capped by "one NOCStruct with certs for your fabric, plus 254 other NOCStructs with just a fabric index".  We don't really have a way to express that in the XML, and I'm not sure what someone would do with that information anyway...